samples/getting-started/README.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started with the Azure Quantum Optimization Service #
```

Comment:
Brand is Microsoft QIO, as I understand it. 
Will calling it Azure here and Microsoft QIO a few sentences down cause confusion?

Reply:
Thanks for raising this! I believe that this line should be consistent with branding, as this is the README for Azure Quantum optimization service samples all-up, not only the Microsoft QIO provider.